Uni_life_tracker/app/src/main/res/layout/activity_main.xml

```diff
@@ -25,7 +25,7 @@
     <TextView
             android:layout_width="377dp"
             android:layout_height="200dp"
-            android:text="@string/uni_life_tracker"
+            android:text="@string/app_name"
```

Comment:
Minor detail: but would it be possible to change the font to roboto_mono and roboto_mono_bold where needed